### PR TITLE
[13.0][ADD] maintenance_request_employee: New module

### PR DESCRIPTION
**Pull request by victoralmau**
New module

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT41735

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
Code review :+1:

**Comment by victoralmau** [comment]
Ping @pedrobaeza

File: maintenance_request_employee/views/maintenance_request_views.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            name="inherit_id"
+            ref="hr_maintenance.maintenance_request_view_kanban_inherit_hr"
+        />
+        <field name="model">maintenance.request</field>
+        <field name="arch" type="xml">
+            <xpath
+                expr="//span[@t-if='record.department_id.raw_value']"
+                position="after"
+            >
+                <span t-if="record.employee_ids.raw_value"><field
```
Comment (pedrobaeza) [inline maintenance_request_employee/views/maintenance_request_views.xml]: Better to show the avatars, isn't it?

**Review by pedrobaeza — COMMENTED** [review]
Please put this icon as a combination of `maintenance` and `hr` ones.

icon
icon

**Comment by victoralmau** [comment]
Thanks, changes done.

File: maintenance_request_employee/views/maintenance_request_views.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                <div t-if="record.employee_ids.raw_value">
+                    <t
+                        t-foreach="record.employee_ids.raw_value.slice(0,11)"
+                        t-as="employee"
+                    >
+                        <img
+                            class="oe_kanban_avatar o_image_24_cover"
+                            t-att-src="kanban_image('hr.employee', 'image_128', employee)"
+                            t-att-data-member_id="employee"
+                            alt="Employee"
```
Comment (pedrobaeza) [inline maintenance_request_employee/views/maintenance_request_views.xml:60]: Can you put here the employee name?

Comment (victoralmau) [reply maintenance_request_employee/views/maintenance_request_views.xml:60]: I think it is not possible because we are iterating ids and not recordset.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**